final_task/rss_reader/rss_reader.py

```diff
@@ -0,0 +1,183 @@
+import argparse
+from typing import Union
+
+import feedparser
+import logging
+import json
+import html
+import sys
+import datetime
+import os
+
+__VERSION__ = '3.5'
+ARGS = None
+LIMIT = 3
+LINKS = []
+PRINT_SETTING = {
+    'feed': [
+        'title'
+    ],
+    'entries': [
+        'title',
+        'published',
+        'link',
+        'summary'
+    ]
+}
```

Comment:
IN genereal it is better to avoid global variables whenever you can. You could just pass all of them as arguments to the corresponfing functions instead